rllib/core/models/base.py

```diff
@@ -195,7 +202,8 @@ class Encoder(Model, abc.ABC):
     Similarly, their output_spec contains the latent space dimensions.
     Encoders can be recurrent, in which case the state should be part of input- and
     output_specs. The latents that are produced by an encoder are fed into subsequent
-    heads.
+    heads. Any implementation of Encoder should also be callable. This should be done
+    by also inheriting from a framework-specific model base-class, s.a. TorchModel.
```

Comment:
Another thing Jun pointed out - simply subclassing encoder is not enough, it won't make the model callable from the RLModule. This needs to be pointed out.